src/utils/version.rs

```diff
@@ -14,11 +14,11 @@ const ONE_DAY: u64 = ONE_HOUR * 24;
 
 /// check for newer versions of rover.
 ///
-/// If this fn is run explicitly from a user-facing command, we pass `force` to
+/// If this fn is run explicitly from a user-facing command, we pass `is_foreground` to
 /// check for newer versions, even if we recently checked for updates.
 ///
-/// If `force` is not passed, we check for updates every day at most
-pub fn check_for_update(config: config::Config, force: bool) -> Result<()> {
+/// If `is_foreground` is not passed, we check for updates every day at most
+pub fn check_for_update(config: config::Config, is_foreground: bool) -> Result<()> {
```

Comment:
I don't love the `is_foreground` name. It sounds less obvious what's going on than `force` imho. But I won't block on that at all :)

Reply:
mmm - i'll switch it back.